addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -78,7 +78,21 @@ var Wysiwyg = Widget.extend({
                     {
                         selector: [this.JWEditorLib.DividerNode],
                         breakable: false,
+                        renderbr: true,
                     },
+                    {
+                        selector: [
+                            (node) => {
+                                const attributes = node.modifiers.find(this.JWEditorLib.Attributes);
+
+                                const isCountdown = attributes && attributes.classList.has('s_countdown');
+                                const isTextarea = node instanceof this.JWEditorLib.VElement && node.htmlTag === 'TEXTAREA';
```

Comment:
I'm guessing this will change when `TextAreaNode` is merged?

Reply:
Yes